docs/quickstarts/csharp-quickstart.md

```diff
@@ -0,0 +1,254 @@
+---
+layout: docs
+title: C# Quickstart
+---
+
+<h1 class="page-header">C# Quickstart</h1>
+
+<p class="lead">This guide gets you started with gRPC in C# with a simple working example.</p>
+
+<div id="toc"></div>
+
+## Before you begin
+
+### Prerequisites
+
+* Windows: .NET Framework 4.5+, Visual Studio 2013 or 2015. You may also need the Nuget executable 
+* Linux: Mono 4+, MonoDevelop 5.9+ (with NuGet add-in installed)
+* Mac OS X: Xamarin Studio 5.9+
+
+### How to use Grpc
+
+The example solution in this walkthrough already adds the necessary
+dependencies on Grpc, but this explains how to use Grpc in your projects.
```

Comment:
...dependencies on gRPC. When you create your own projects, you'll need to do the following to set up the appropriate dependencies.

(I'm slightly iffy about this being included in a Quickstart where you don't actually have to do *any* of this to complete the Quickstart itself - it's purely for working on your *own* projects, and the Quickstart is supposed to be a super quick "let's get something up and running" tutorial. Though I'm not sure where we'd put it otherwise..in the repo? On another page? Thoughts?)